docs/user-guide/ecosystem.md

```diff
@@ -0,0 +1,61 @@
+# Polars ecosystem
+
+## Introduction
+
+On this page you can find a non-exhaustive list of libraries and tools that support Polars. As the data ecosystem is evolving fast, more libraries will likely support Polars in the future. One of the main drivers is that Polars makes use of `Apache Arrow` in it's backend.
+
+#### [Apache Arrow](https://arrow.apache.org/)
+
+Apache Arrow enables zero-copy reads of data, meaning that data can be directly accessed in its in-memory format without the need for copying or serialisation. This greatly enhances performance, especially when sharing data between different processes or systems. Using Apache Arrow, Polars is compatible with a wide range of libraries that also make use of Apache Arrow, like Pandas and Pyspark.
+
+### Data visualisation
+
+#### [hvPlot](https://hvplot.holoviz.org/)
+
+hvPlot is available as the default plotting backend for Polars making it simple to create interactive and static visualisations. You can use hvPlot by using the feature flag `plot` during installing.
+
+```python
+pip install 'polars[plot]'
+```
+
+#### [Matplotlib](https://matplotlib.org/)
+
+Matplotlib is a comprehensive library for creating static, animated, and interactive visualizations in Python. Matplotlib makes easy things easy and hard things possible.
```

Comment:
How does polars interact with matplotlib?

Reply:
There is a pending PR for a page about data visualisations from @braaannigan . My idea is to link to that page or show the plots on this page. I have opened an issue to open the discussion. That is also why this PR is in draft

Reply:
@r-brink I'll try to get my PR fixed in the next couple of days

Reply:
@braaannigan that would be great. I will update the PR with links to the section on your visualisation page!

Reply:
@r-brink Hopefully can get this merged soon #13052